### PR TITLE
Update slackbot project

### PR DESCRIPTION
**Pull request by strickvl**

### CODE REVIEW

File: langchain-llamaindex-slackbot/src/requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ langchain==0.0.125
 openai==0.27.2
 slack-bolt==1.16.2
 slack-sdk==3.20.0
--e  git+https://github.com/zenml-io/zenml.git@develop#egg=zenml
+zenml==0.40.0
```
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/requirements.txt:5]: Does this work? I tried with 0.39.1 but it didnt work due to some internal error

Comment (strickvl) [reply langchain-llamaindex-slackbot/src/requirements.txt:5]: Unsure. Will have to wait for 0.40.1 to test. This is just a draft for now.

Comment (strickvl) [reply langchain-llamaindex-slackbot/src/requirements.txt]: It works.

File: langchain-llamaindex-slackbot/README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ After the installation is completed, you directly run the pipeline locally
 right away.
 
 ```bash
-python run.py
+python src/run.py
```
Comment (fa9r) [inline langchain-llamaindex-slackbot/README.md:80]: Why is this necessary now? Or was this always broken?

File: langchain-llamaindex-slackbot/src/slackbot_utils.py [hunk 3]
```diff
@@ -58,7 +58,8 @@ def connect_to_zenml_server():
 
 def get_vector_store():
     """Get the vector store from the pipeline."""
-    pipeline = get_pipeline(pipeline=PIPELINE_NAME, version=9)
+    # gets the most recent pipeline version
+    pipeline = get_pipeline(pipeline=PIPELINE_NAME).versions[0]
     our_run = pipeline.runs[0]
     print("Using pipeline: ", pipeline.model.name, "v", pipeline.model.version)
     print("Created on: ", pipeline.model.updated)
```
Comment (fa9r) [inline langchain-llamaindex-slackbot/src/slackbot_utils.py:65]: Version accessing is really confusing, I think versions are sorted by creation time, not by time of their last runs, so `versions[0]` might not always work as expected. However, `pipeline.runs[0]` should always give you the last run under that pipeline name at least.

```suggestion
    pipeline = get_pipeline(pipeline=PIPELINE_NAME)
    our_run = pipeline.runs[0]
    print("Using pipeline: ", pipeline.name)
```

**Review by fa9r — CHANGES_REQUESTED** [review]

File: langchain-llamaindex-slackbot/README.md (hunk 2 above)
Comment (strickvl) [reply langchain-llamaindex-slackbot/README.md:80]: Was always broken / incorrectly documented. Only came across it while redeploying.

File: langchain-llamaindex-slackbot/src/slackbot_utils.py (hunk 3 above)
Comment (strickvl) [reply langchain-llamaindex-slackbot/src/slackbot_utils.py:65]: In testing, this actually worked, but your way seems less confusing so let's go with that. In any case it's an improvement on manually specifying the version (which we were previously doing).

**Review by fa9r — APPROVED** [review]

**Review by htahir1 — APPROVED** [review]
